tests/org.palladiosimulator.dataflow.confidentiality.analysis.tests/src/org/palladiosimulator/dataflow/confidentiality/analysis/constraint/data/ConstraintData.java

```diff
@@ -0,0 +1,48 @@
+package org.palladiosimulator.dataflow.confidentiality.analysis.constraint.data;
+
+import java.util.List;
+import java.util.Map;
+
+import org.palladiosimulator.dataflow.confidentiality.analysis.sequence.entity.AbstractActionSequenceElement;
+import org.palladiosimulator.dataflow.confidentiality.analysis.sequence.entity.CharacteristicValue;
+import org.palladiosimulator.dataflow.confidentiality.analysis.sequence.entity.DataFlowVariable;
+import org.palladiosimulator.dataflow.confidentiality.analysis.sequence.entity.pcm.AbstractPCMActionSequenceElement;
+
+import de.uka.ipd.sdq.identifier.Identifier;
+
+public class ConstraintData {
```

Comment:
Could this be a record?

Reply:
I don't think it is necessary to use a record here, as the attributes of that class are only used internally